x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.tsx

```diff
@@ -50,18 +50,22 @@ export const TrainedModelHealthPopover: React.FC<InferencePipeline> = (pipeline)
 
   const { pipelineName } = pipeline;
 
-  const actionButton = (
+  const wrapperButton = (
```

Comment:
shouldn't we define this like 

```const WrapperButton: React.FC<WrapperButtonProps> = ({ children }) => (
  <EuiButtonEmpty
    data-test-subj="enterpriseSearchTrainedModelHealthPopoverButton"
    iconSide="right"
    flush="both"
    iconType="boxesHorizontal"
   onClick={() => setIsPopOverOpen(!isPopOverOpen)}
  >
    {children}
  </EuiButtonEmpty>
);
```

Reply:
So I've tried to implement this but the tooltip stopped working. As far as I can see on the DOM, the output is identical. I'm at a loss as to why it doesn't work.
Here's a draft PR if you want to see. Maybe you'll notice something I haven't.
https://github.com/elastic/kibana/pull/201409

Reply:
If I replace `WrapperElement` on line 215 with a `EuiButtonEmpty` directly, the tooltip works

Reply:
I was able to fix it by wrapping it in a `span`! Apparently the element being generated by a function was preventing the tooltip's onFocus to properly inherit, adding the `span` allows the `span` to have the functionality in its place